File: AppBar/app/src/main/java/com/example/appbar/MainActivity.java
package com.example.appbar;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import android.os.Bundle;
import android.view.Menu;

public class MainActivity extends AppCompatActivity {

    Toolbar toolbar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        toolbar = findViewById(R.id.toolbar1);
        toolbar.inflateMenu(R.menu.main_menu);
    }

	// oncreate 

    public boolean onCreateOptionsmenu(Menu menu) {
        getMenuInflater().inflate(R.menu.main_menu, menu);
        return true;
    }
}

File: ScreenSlideAnimation/app/src/main/java/com/example/screenslideanimation/MainActivity.kt
package com.example.screenslideanimation

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import androidx.viewpager.widget.ViewPager

class MainActivity : AppCompatActivity() {

    var viewPager: ViewPager? = null
    var customSwipeAdapter: CustomSwipeAdapter? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        viewPager = findViewById<View>(R.id.viewpager) as ViewPager
        customSwipeAdapter = CustomSwipeAdapter(this)
        viewPager?.adapter = customSwipeAdapter
    }
}